File: RMSwitchDemo/RMSwitchDemo/ViewController.m
//
//  ViewController.m
//  RMSwitchDemo
//
//  Created by Rookieme on 2018/12/28.
//  Copyright © 2018 Rookieme. All rights reserved.
//

#import "ViewController.h"
#import "RMSwitch/RMSwitch.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
//     Do any additional setup after loading the view, typically from a nib.
    self.view.backgroundColor = [UIColor whiteColor];
    RMSwitch *rmswitch = [[RMSwitch alloc] initWithFrame:CGRectMake(100, 100, 30, 10)];
    rmswitch.backgroundColor = [UIColor orangeColor];
    [self.view addSubview:rmswitch];
}


@end

File: RMNavigation/RMNavigation/ViewController.swift
//
//  ViewController.swift
//  RMNavigation
//
//  Created by rookieme on 2018/12/11.
//  Copyright © 2018 rookieme. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        view.backgroundColor = UIColor.white
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        let vc = UIViewController()
        vc.view.backgroundColor = UIColor.orange
        navigationController?.pushViewController(vc, animated: true)
    }

    override func didMove(toParent parent: UIViewController?) {
        super.didMove(toParent: parent)
        print("come didMove")
    }
    
    override func willMove(toParent parent: UIViewController?) {
        super.willMove(toParent: parent)
        print("come willMove")
    }

}

